libraries/shared/factories/storage/README.md

```diff
@@ -1,89 +1,98 @@
 # Watching Contract Storage
 
-One approach VulcanizeDB takes to caching and indexing smart contracts is to ingest raw contract storage values.
-Assuming that you are running an ethereum node that is writing contract storage changes to a CSV file, VulcanizeDB can parse them and persist the results to postgres.
+Using the `extractDiffs` command VulcanizeDB can ingest raw contract storage values and store them in the `public.storage_diffs` table in postgres, allowing for transformation by the `execute` command.
 
 ## Assumptions
 
-The current approach for caching smart contract storage diffs assumes that you are running a node that is writing contract storage diffs to a CSV file.
-The CSV file is expected to have 5 columns: contract address, block hash, block number, storage key, storage value.
+We have [a branch of go-ethereum (a.k.a geth)](https://github.com/makerdao/go-ethereum) that enables running a node that provides storage diffs via a websocket, emitting them as nodes are added to the chain, and is required for using `extractDiffs`. That command will continuously read from the websocket and persist the results to postgres, in the `public.storage_diffs` table to be transformed.
```

Comment:
I think instead of "emitting them as nodes are added to the chain" we could say "as blocks are added" or "as transactions are applied on"